frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -109,7 +109,7 @@ export function SDKs({
                             allowClear
                             onChange={(v) => setSourceFilter(v)}
                             options={sourceOptions}
-                            placeholder="Select a source type"
+                            placeholder="Select a framework type"
```

Comment:
Heads up I renamed this. Personally I didnt understand what a "source" is and thought framework is a better term

Reply:
This makes the button text wrap and looks bad, can we keep it saying "source"?